src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -210,7 +211,8 @@ def default_api_version(self):
         ResourceType.MGMT_APPSERVICE: '2020-09-01',
         ResourceType.MGMT_IOTHUB: '2021-03-31',
         ResourceType.MGMT_ARO: '2020-04-30',
-        ResourceType.MGMT_DATABOXEDGE: '2019-08-01'
+        ResourceType.MGMT_DATABOXEDGE: '2019-08-01',
+        ResourceType.MGMT_CUSTOMLOCATION: '2021-03-15-preview'
```

Comment:
do we need to support other profiles, e.g. 2020-09-01-hybrid?

Reply:
I only need to use the latest, our use case is doing a GET call to ExtendedLocation to make sure the resource exists, before using it in AppService